rental.rb

```diff
@@ -0,0 +1,15 @@
+class Rental
+  attr_accessor :date
+
```

Comment:
- [ ] You're yet to add the getters and setters for book and person as requested by the first Code Reviewer.